CliClient/tests/services_SearchFuzzy.js

```diff
@@ -141,5 +141,29 @@ describe('services_SearchFuzzy', function() {
 		expect(rows.map(r=>r.id)).toContain(n5.id);
 	}));
 
+	it('should work with wild card searches', asyncTest(async () => {
+		let rows;
+		const n1 = await Note.save({ title: 'abc def' });
+		const n2 = await Note.save({ title: 'abcc ghi' });
+		const n3 = await Note.save({ title: 'abccc jkl' });
+		const n4 = await Note.save({ title: 'abcccc mno' });
+		const n5 = await Note.save({ title: 'abccccc pqr' });
+		const n6 = await Note.save({ title: 'abcccccc xyz' });
+		const n7 = await Note.save({ title: 'abccccccc 123' });
+		const n8 = await Note.save({ title: 'wxy zzz' });
```

Comment:
Wouldn't a minimal test in this case be just n1, n2 and n8? Like I'm not sure having n3-7 makes a difference for the test and in general the smaller the test the better.

Reply:
It was to make the test fail in the previous implementation. I'll make it shorter.